optuna/multi_objective/trial.py

```diff
@@ -276,7 +272,7 @@ def __init__(self, n_objectives: int, trial: FrozenTrial):
         self.n_objectives = n_objectives
         self._trial = trial
 
-        self.values = tuple(trial.intermediate_values.get(i) for i in range(n_objectives))
+        self.values = [trial.intermediate_values.get(i) for i in range(n_objectives)]
```

Comment:
Could you elaborate a bit on this change?

Reply:
During try-and-error I found `values` (and `_get_values`) of `FrozenTrial` is a list of float. My approach was to add `_get_values` to `FrozenMultiObjectiveTrial` so I updated the type hint for `self.values` from tuple to list.

Now I changed the approach to use self.values directly (also related. https://github.com/optuna/optuna/pull/3144#discussion_r782098716) and reverted the change in 2371b45.